libs/mobilePayments.js

```diff
@@ -1,56 +1,59 @@
-const nconf = require('nconf');
-const request = require('request');
-const moment = require('moment');
+import nconf from 'nconf';
+import request from 'request';
+import moment from 'moment';
 
 const BASE_URL = nconf.get('BASE_URL');
 
-const api = {};
-
-api.cancelSubscriptionForUser = function cancelSubscriptionForUser (habitrpgUsers, user, platform) {
-    return new Promise((resolve, reject) => {
-      request.get(`${BASE_URL}/iap/${patform}/subscribe/cancel`, {
+const cancelSubscriptionForUser = function cancelSubscriptionForUser (habitrpgUsers, user, platform) {
+  return new Promise((resolve, reject) => {
+    request.get(`${BASE_URL}/iap/${platform}/subscribe/cancel`, {
       qs: {
         noRedirect: 'true',
       },
       headers: {
         'x-api-user': user._id,
         'x-api-key': user.apiToken,
       },
-      }, (habitError, habitResponse, body) => {
-          if (habitResponse.statusCode === 401) {
-            return habitrpgUsers.update(
-                {
-                  _id: user._id,
-                },
-                {
-                  $set: {
-                    'purchased.plan.dateTerminated': Date(),
-                  },
-                });
-          }
-          if (!habitError && habitResponse.statusCode === 200) {
-            return resolve();
-          }
+    }, (habitError, habitResponse, body) => {
+      if (habitResponse.statusCode === 401) {
+        return habitrpgUsers.update(
+          {
+            _id: user._id,
+          },
+          {
+            $set: {
+              'purchased.plan.dateTerminated': Date(),
```

Comment:
Does this work? We've always done `new Date()` with these actions, storing the `ISODate` function. Will Mongo parse it out if we just send the timestamp string?